### PR TITLE
[Stream.cz] add subtitle support

### PR DESCRIPTION
**Pull request by oskar456**
### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [adding new extractor tutorial](https://github.com/rg3/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/rg3/youtube-dl#youtube-dl-coding-conventions) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [ ] Bug fix
- [x] Improvement
- [ ] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information

This adds subtitle support for Stream.cz. Also tests are fixed (they probably somehow changed the video files).

### CODE REVIEW

File: youtube_dl/extractor/streamcz.py [hunk 1]
```diff
@@ -93,4 +93,15 @@ def _real_extract(self, url):
             'description': data.get('web_site_text'),
             'duration': int_or_none(data.get('duration')),
             'view_count': int_or_none(data.get('views')),
+            'subtitles': self.extract_subtitles(data.get('subtitles_srt')),
         }
+
+    @staticmethod
+    def _get_subtitles(subs):
```
Comment (dstftw) [inline youtube_dl/extractor/streamcz.py]: What's the point of this?

Comment (oskar456) [inline youtube_dl/extractor/streamcz.py]: It's the method called by `self.extract_subtitles()` common method when subtitles are actually requested by user.

In this case, it's probably unnecessary.

Comment (dstftw) [inline youtube_dl/extractor/streamcz.py]: This methods is only used when subtitles' extraction depends on additional expensive work, i.e. network I/O. In this case subtitles are already extracted along with formats and using this method does not make any sense.

Comment (oskar456) [inline youtube_dl/extractor/streamcz.py]: Thanks for guidance. I've reworked it so it does the subtitle extracting along the main `_real_extract` method.

**Pull request merged**